policyengine_us/variables/household/demographic/tax_unit/tax_unit_stillborn_parent.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_us.model_api import *
+
+
+class tax_unit_stillborn_parent(Variable):
+    value_type = int
```

Comment:
should this be a bool? not sure we need the number of stillborns

Reply:
you suggested it to be an int before

> It looks like it's one for each stillborn so make an int

Reply:
The tax form allows for a multiplication by stillborn certificates: 
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/0102d3e3-d866-4eb5-898d-ed4903402c6f)

Reply:
Thanks, my mistake. In that case please rename the variable to `tax_unit_stillbirths`